src/__Tests__/calculate.test.js

```diff
@@ -0,0 +1,211 @@
+import calculate from '../logic/calculate';
+
```

Comment:
- [ ] Jest snapshots are missing in your test. Kindly ensure that Jest snapshots are used for testing **all** the components.